easybuild/easyconfigs/d/DALTON/DALTON-2016.2-gimkl-2017b.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'CMakeMake'
+
+name = 'DALTON'
+version = '2016.2'
+
+homepage = 'https://www.daltonprogram.org/'
+description = """
+The Dalton code is a powerful tool for a wide range of molecular
+properties at different levels of theory.
+Any published work arising from use of one of the Dalton2016 programs
+must acknowledge that by a proper reference,
+https://www.daltonprogram.org/www/citation.html.
+
+This is a OpenMP + MPI hybrid build.
+"""
+
+toolchain = {'name': 'gimkl', 'version': '2017b'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+# Dalton is licensed software. Need to download by hand.
+sources = ['%(name)s%(version)s-Source.tar']
```

Comment:
@akesandgren Now open source? https://gitlab.com/dalton/dalton/tree/Dalton2016.2

Reply:
Apparently yes... will fix.

Reply:
Not that easy. They assume recursive git clone...